### PR TITLE
chore(ci): Re-enable integration_test

### PR DESCRIPTION
**Pull request by dwindsor**
Signed-off-by: David Windsor <dwindsor@secureworks.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area cmd

> /area pkg

> /area docs

/area tests

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by FedeDP** [comment]
>  CircleCI Pipeline — Unable to parse YAML 

Sorry what? :laughing:

File: .circleci/config.yml [hunk 1]
```diff
@@ -51,9 +51,9 @@ jobs:
       - run:
           name: Test
           command: docker exec alpine_sh make test
-      # - run:
-      #     name: Integration tests
-      #     command: docker exec alpine_sh make integration_test
+       - run:
```
Comment (FedeDP) [inline .circleci/config.yml:54]: Empty space before `- run` breaking circleci!
Same for the following 2 lines.

File: Makefile [hunk 2]
```diff
@@ -124,7 +124,7 @@ integration_test: $(test_configs)
 
 .PHONY: $(test_configs)
 $(test_configs): ${driverkit}
-	${driverkit} docker -c $@ --builderimage auto:latest -l debug --timeout 600
+	${driverkit} docker -c $@ --builderimage auto:master -l debug --timeout 600
```
Comment (FedeDP) [inline Makefile:127]: It makes sense to use master tag for integration tests IMHO.

**Comment by FedeDP** [comment]
> ERROR: "tp_names" [/tmp/driver/falco.ko] undefined! 

That's an issue with libs master :/

**Comment by FedeDP** [comment]
Issue will be fixed by #209 that is also (of course) backward compatible.

**Comment by FedeDP** [comment]
Rebased and force-pushed to trigger CI that should now work :)

**Comment by FedeDP** [comment]
It seems like buster builder image was only pushed for x86_64 :/

**Comment by FedeDP** [comment]
Yep because gcc-4.9 cannot be found for debian buster on arm64. 
I am going to fix it in another PR.

**Comment by FedeDP** [comment]
The PR that fixes the issue is #210 . Unfortunately i couldn't find any gcc-4.9 for arm64.

**Comment by dwindsor** [comment]
Closing in favor of #211

**Pull request closed without merging**